### ISSUE TITLE
Add error message when __init__.py is missing in a root package

### ISSUE THREAD

**Issue by kamadorueda** [seddonym/import-linter#93]
The tool is awesome and works very well, is just that it fails silently on this condition which gives a bad experience

![image](https://user-images.githubusercontent.com/47480384/104591070-327bd400-563a-11eb-86e6-e49e80c4c995.png)

**Comment by seddonym** [comment]
Thanks for the bug report. This is related to [this issue](https://github.com/seddonym/grimp/issues/53) on a package that Import Linter depends on, which I also maintain. It is on my list to do, pull requests also welcomed!

**Comment by kamadorueda** [comment]
I think the tool is definitely a requirement for good-architectured python systems

Have you thought on adding a graph visualizer?

- this is our rocky-river, modules: ![image](https://user-images.githubusercontent.com/47480384/104593249-64db0080-563d-11eb-9069-9db73636937f.png)
- by files: ![image](https://user-images.githubusercontent.com/47480384/104593266-6f959580-563d-11eb-89f0-3a7cdc346580.png)

we can plot the networkx from grimp with graphviz and produce such graphs

I think I can work in PRs for it

**Comment by seddonym** [comment]
Very nice!

You might be interested in [Impulse](https://github.com/seddonym/impulse), the sister package of Import Linter, which is a tool for drawing these kinds of diagrams. I wanted to keep Import Linter focused on the linting but they both depend on a shared package [Grimp](https://github.com/seddonym/grimp) which builds the networkx graph from the Python code.

**Comment by seddonym** [comment]
I've merged this issue to the master branch of Grimp. @kamadorueda would you mind installing that and see if you feel it has improved things? I'll then release Grimp and Import Linter with the latest version.

**Comment by kamadorueda** [comment]
works awesome

---

Before:

```
kamado:~/Documents/fluidattacks/product/skims/skims$ lint-imports --config ../setup.imports.cfg 

kamado:~/Documents/fluidattacks/product/skims/skims$ echo $?
1
```

after

```
kamado:~/Documents/fluidattacks/product/skims/skims$ lint-imports --config ../setup.imports.cfg 
Package aws appears to be a 'namespace package' (see PEP 420), which is not currently supported. If this is not deliberate, adding an __init__.py file should fix the problem.
kamado:~/Documents/fluidattacks/product/skims/skims$ echo $?
1
```

after adding the `__init__.py`:

```
kamado:~/Documents/fluidattacks/product/skims/skims$ lint-imports --config ../setup.imports.cfg 
=============
Import Linter
=============

---------
Contracts
---------

Analyzed 149 files, 507 dependencies.
-------------------------------------

Direct Acyclic Graph KEPT

Contracts: 1 kept, 0 broken.
kamado:~/Documents/fluidattacks/product/skims/skims$ echo $?
0
```

**Comment by seddonym** [comment]
This is now fixed in the new release, Import Linter 1.2.1. Thanks again!
